randovania/games/prime1/json_data/Tallon Overworld.txt

```diff
@@ -319,14 +319,14 @@ Extra - aabb: [-286.18207, 307.2683, -70.64886, 66.35881, 622.5101, 151.42361]
           All of the following:
               Scan Visor
               Any of the following:
-                  # https://www.youtube.com/watch?v=O9tQK-WsqaI
+                  # https://youtu.be/X1rAc0AROx8
                   Combat/Scan Dash (Intermediate)
                   Space Jump Boots and Combat/Scan Dash (Beginner)
           All of the following:
-              # https://youtu.be/1RHKJnJk-O0
+              # https://youtu.be/X1rAc0AROx8?t=19
               No Gravity Suit and Slope Jump (Advanced) and Gravityless Underwater Movement (Advanced) and Enabled Allow Dangerous Gravity Suit Logic
           All of the following:
-              # https://youtu.be/5h_hb9I45sI
+              # https://youtu.be/X1rAc0AROx8?t=8
```

Comment:
How come these got recorded twice? Did I make a clerical error on the spreadsheet?

Reply:
I believe this was do to Sam recording the 3 NSJ methods in one video and I updated all 3 methods. Nothing wrong with original video so I can revert it back.

Reply:
leave as is